contrib/bradfitz/gomemcache/memcache/memcache.go

```diff
@@ -88,6 +91,9 @@ func (c *Client) Add(item *memcache.Item) error {
 func (c *Client) CompareAndSwap(item *memcache.Item) error {
 	span := c.startSpan("CompareAndSwap")
 	err := c.Client.CompareAndSwap(item)
+	span.SetTag("item.key", item.Key)
+	span.SetTag("item.value", item.Value)
```

Comment:
I think that we'll want these off by default. They might contain sensitive information that other users might not want to share. We can chose two ways to address this:

1. Add an option `WithValueTags` and when this option is set, display the tags, leaving it up to the user.
2. Not add them at all.

What do you think? Feel free to chose any one which suits you best.

Reply:
I think making it an option is better than leaving it impossible. I'd like to select the first option.

Reply:
As you prefer. Whichever one you like!

Reply:
Done :)